package.json

```diff
@@ -29,6 +32,10 @@
     "solidity"
   ],
   "license": "MIT",
+  "peerDependencies": {
+    "@forge/std": "github:foundry-rs/forge-std#e8a047e3f40f13fa37af6fe14e6e06283d9a060e",
+    "@prb/test": "0.6.4"
+  },
```

Comment:
Should be removed

Reply:
changed your mind about this? 😅

https://github.com/PaulRBerg/prb-math/pull/206#discussion_r1375931024